src/app/components/SatButtons/index.tsx

```diff
@@ -38,6 +31,13 @@ function SatButtons({ onClick, disabled }: Props) {
         fullWidth
         disabled={disabled}
       />
+      <Button
+        icon={<SatoshiV2Icon className="w-4 h-4" />}
+        label="25K ⚡"
```

Comment:
Why is it a capital "K" here?
https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes :)